projects/plugins/jetpack/modules/simple-payments.php

```diff
@@ -0,0 +1,17 @@
+<?php // phpcs:ignore Squiz.Commenting.FileComment.Missing
+/**
+ * Simple Payments module loader.
+ *
+ * @package automattic/jetpack
+ */
+
+require_once __DIR__ . '/../packages/paypal-payments/class-simple-payments.php';
```

Comment:
That won't work on production since the required package will not live in this directory in the production version of the plugin. 

Since the package will be autoloaded, you do not have to worry about requiring the file.

Reply:
I've removed the `require_once` statement in [0169271](https://github.com/Automattic/jetpack/pull/43413/commits/0169271ff88d91506a7733109c5dc80c488064fd).

That said, do I need to add a `use` statement to import the correct one? After removing the `require_once` my IDE is showing it using the `Jetpack_Simple_Payments::get_instance();` from the deprecated class.

Reply:
I've added that `use` statement to get it to use the correct class in [0b47eca](https://github.com/Automattic/jetpack/pull/43413/commits/0b47eca3388467ff407304dbee4dd219de023844).